src/aggregate/aggregate_exec.c

```diff
@@ -1152,8 +1152,19 @@ int RSCursorCommand(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
     long long count = 0;
     if (argc > 5) {
       // e.g. 'COUNT <timeout>'
+      // Verify that the 4'th argument is `COUNT`.
+      const char *count_str = RedisModule_StringPtrLen(argv[4], NULL);
+      if (strcasecmp(count_str, "count") != 0) {
+        char err[128];
+        sprintf(err, "Unsupported argument `%s`", count_str);
+        RedisModule_ReplyWithError(ctx, err);
+        RedisModule_EndReply(reply);
+        return REDISMODULE_OK;
+      }
+
       if (RedisModule_StringToLongLong(argv[5], &count) != REDISMODULE_OK) {
         RedisModule_ReplyWithError(ctx, "Bad value for COUNT");
+        RedisModule_EndReply(reply);
```

Comment:
1. Can we test this?
Maybe adding a test like:
`res, cid = env.cmd('FT.CURSOR', 'READ', 'idx', str(cid), 'count', '2.3')`

2. What is the expected error if `COUNT < 0` ?

Reply:
1. Good idea. Test added.
2. 
```
127.0.0.1:6379> FT.AGGREGATE idx * withcursor count 2.3
(error) Bad arguments for COUNT: Could not convert argument to expected type
```
Test added as well